src/shadowbox/server/manager_metrics.ts

```diff
@@ -38,4 +41,18 @@ export class PrometheusManagerMetrics implements ManagerMetrics {
     }
     return {bytesTransferredByUserId: usage};
   }
+
+  async getOutboundByteTransfer(accessKeyId: string, windowHours: number):
+      Promise<DataUsageByUser> {
+    let bytesTransferred = 0;
+    const result = await this.prometheusClient.query(
+        `sum(increase(shadowsocks_data_bytes{dir=~"c<p|p>t|>p<",access_key="${accessKeyId}"}[${
+            windowHours}h])) by (access_key)`);
+    if (result && result.result[0] && result.result[0].metric['access_key'] === accessKeyId) {
+      bytesTransferred = Math.round(parseFloat(result.result[0].value[1]));
+    }
+    const bytesTransferredByUserId = {};
+    bytesTransferredByUserId[accessKeyId] = bytesTransferred;
+    return {bytesTransferredByUserId};
```

Comment:
Why do we wrap it in another object?  Could we

`return {accessKeyId: bytesTransferred}` or even just `return bytesTransferred` directly?